src/services/IconService.js

```diff
@@ -71,26 +71,31 @@ export class IconService {
 	}
 
 	/**
-	  *
-	  * @param {*} idOrUrlOrBase64
+	  * Returns the {@see IconResult} specified by ID, Url or a base64 encoded content-string
+	  * @param {string} idOrUrlOrBase64
+	  * @returns {IconResult|null}
 	*/
 	getIconResult(idOrUrlOrBase64) {
 		if (idOrUrlOrBase64 === this._default.base64) {
 			return this._icons.find(iconResult => iconResult.matches(Svg_Marker_Name));
 		}
-		return this.isLocal(idOrUrlOrBase64) ? this._icons.find(iconResult => iconResult.base64 === idOrUrlOrBase64) : this._icons.find(iconResult => iconResult.matches(idOrUrlOrBase64));
+
+		const findLocal = (base64) => this._icons.find(iconResult => iconResult.base64 === base64) || null;
+		const findRemote = (idOrUrl) => this._icons.find(iconResult => iconResult.matches(idOrUrl)) || null;
```

Comment:
We should always use the "nullish coalescing operator" instead of the "logical or operator" in such cases now 🤓